src/Security/Serializer.php

```
@@ -0,0 +1,833 @@
+     * @throws UnexpectedValueException On type mismatch
+     */
+    public static function toObject(string $payload, string $className, string $format = Format::PHP): object
+    {
+        $data = null;
+        if ($format === Format::PHP) {
+            $data = self::fromPhp($payload, [$className]);
+        } elseif ($format === Format::LEGACY) {
+            $data = self::fromLegacy($payload, [$className]);
+        } else {
+            throw new RuntimeException('Objects only supported in PHP/Legacy formats');
+        }
```

PHPMD (2.15.0)

[error] The method toObject uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)